src/floatingTextInput/index.tsx

```diff
@@ -93,12 +94,20 @@ const FloatingTextInput = ({
           <Text
             style={{
               ...styles.label,
+              ...styles.animatedStyle, left: icon ? 30 : 15,
```

Comment:
- [ ] The low label mentioned in issue #3 is because of this code.
- [ ] kindly remove the `...styles.animated` value because we have already declared the top value . so make it only
```js
<Text
            style={{
              ...styles.label,
              ...labelStyle,
                left: icon ? 30 : 15,
              color: error ? 'red' : labelColor,
            }}>
            {label}
          </Text>
```